src/components/TimestampForm.js

```diff
@@ -47,20 +55,46 @@ const TimestampForm = ({ history }) => {
       `results?digests=${digests.toString()}&names=${names.toString()}&timestamp=true`
     );
   };
+  const handleExpandDetails = () => {
+    setDetailsOpen(!detailsOpen);
+  };
   return (
     <Card>
       <Form>
         <Title>Timestamp</Title>
         <Description>
-          The timestamp service allows you to create a Proof-of-Existence of a
-          given file. A digital signature (digest) of each file is calculated
-          and sent to the Dcrtime server which hourly will calculate the merkle
-          root for all digests collected in the previous 60 minutes and store
-          its value in the blockchain. <br /> The files you submit below will be
-          verified against Dcrtime to find out if their digests are already in
-          there or not. If the digest is not in Dcrtime, it will be uploaded and
-          should be anchored within the next hour.
+          This free service uses the Decred blockchain to time-anchor arbitrary
+          files, which demonstrates a particular file existed at or before the
+          time it was anchored. A hash of each submitted file is calculated and
+          sent to a dcrtime server, which aggregates these hashes, organizes
+          them into a merkle tree, hashes that tree down to a merkle root, and
+          anchors that merkle root in the Decred blockchain once an hour.
         </Description>
+        <Expandable
+          style={{ maxWidth: "600px" }}
+          expanded={detailsOpen}
+          triggerComponent={
+            <TechnicalDetailsButton onClick={handleExpandDetails}>
+              {detailsOpen ? "Hide" : "Show"} Technical Details{" "}
+            </TechnicalDetailsButton>
+          }
+        >
+          <Description style={{ padding: "1em" }}>
+            This service, dcrtime and Decred use the sha256 hash function.
```

Comment:
it is missing a comma here:

`These service, dcrtime and decred, use the sha256`